src/checks/prettier.ts

```diff
@@ -29,6 +29,11 @@ export default async function prettierCheck() {
       // File from create-react-app not matching our formatting
       // https://github.com/facebook/create-react-app/blob/fdbde1f3c256b43d5386b5ae3a75083dbd8f0aff/packages/cra-template/template/src/reportWebVitals.js
       /src[/\\]reportWebVitals\.js$/,
+      // File from create-next-app not matching our formatting
+      // https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/default/pages/api/hello.js
```

Comment:
Need the link with the actual last commit for this file (instead of just pointing to the latest version in the `canary` branch, like this link does)

I'll do this too.